commands/pull_request.go

```diff
@@ -35,6 +36,9 @@ pull-request -i <ISSUE>
 		If multiple <MESSAGE> options are given, their values are concatenated as
 		separate paragraphs.
 
+	-C, --commit <COMMIT>
```

Comment:
Perhaps the expanded flag name could also be `--reuse-message` to match `git commit`?